cluster-autoscaler/cloudprovider/huaweicloud/huaweicloud_cloud_provider.go

```diff
@@ -65,17 +66,19 @@ func newCloudProvider(opts config.AutoscalingOptions, do cloudprovider.NodeGroup
 	}
 
 	csm := newCloudServiceManager(cloudConfig)
-	sgs, err := csm.ListScalingGroups()
+
+	hcp := &huaweicloudCloudProvider{
+		cloudServiceManager: csm,
+		resourceLimiter:     rl,
+		autoScalingGroup:    make([]AutoScalingGroup, 0),
+	}
+	err = hcp.buildAsgs(do.NodeGroupSpecs)
```

Comment:
We should check if specified the node groups, if not, return an error.

Reply:
yes, will do